sdk/textanalytics/Azure.AI.TextAnalytics/src/TextDocumentBatchStatistics.cs

```diff
@@ -51,5 +52,11 @@ internal TextDocumentBatchStatistics(int documentCount, int validDocumentCount,
         /// </summary>
         [CodeGenMember("TransactionsCount")]
         public long TransactionCount { get; }
+
+        /// <summary>
+        /// Gets the additional properties
+        /// </summary>
+        [CodeGenMember("AdditionalProperties")]
+        internal IDictionary<string, object> AdditionalProperties { get; }
```

Comment:
How do you plan to expose this? Is there a tracking issue?

Reply:
I do not know how yet, which is why I'm keeping it internal for now. Here's an issue to formally track it: https://github.com/Azure/azure-sdk-for-net/issues/31694